include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -73,6 +75,14 @@ struct gap
     //! internal value
     static constexpr integral_type value = 0;
 
+    /*! @name letter values
+     * Static member "letters" that can be assigned to the alphabet or used in aggregate initialization.
+     */
+    //!@{
+    static const gap GAP;
+    static const gap UNKNOWN;
```

Comment:
I regarded it as a special feature of the nucleic acid and aminoacid alphabets... the quality alphabets also don't have it.